src/mui/list/List.js

```diff
@@ -102,10 +106,9 @@ export class List extends Component {
         return this.props.location.pathname;
     }
 
-    refresh = (event) => {
+    refresh = event => {
         event.stopPropagation();
-        this.fullRefresh = true;
-        this.updateData();
```

Comment:
After loading data list component will be rerendered with a new set of data. It doesn't looks like optimal, because component might be rerendered (3 times in one refresh action). But it works now as well.

```
shouldComponentUpdate(nextProps, nextState) {
        if (
            nextProps.isLoading === this.props.isLoading // here will be false when data will be loaded
         && nextProps.width === this.props.width
         && nextProps.version === this.props.version
         && nextState === this.state) {
            return false;
        }
        return true;
    }
```

Maybe I missed smth?